airflow-core/src/airflow/api_fastapi/auth/tokens.py

```diff
@@ -451,7 +451,7 @@ def generate(self, extras: dict[str, Any] | None = None, headers: dict[str, Any]
             del claims["aud"]
 
         if extras is not None:
-            claims.update(extras)
+            claims = extras | claims
```

Comment:
This is to not overwrite anything in claims with values from extra -- i.e. if `extra` includes  "exp" etc we don't want to use it.